Hard/1. Shopping Cart Problem/solutions/damini31_DaminiChauhan/Orders.csv

```diff
@@ -0,0 +1,17 @@
+,Order ID,Order Status,Billing Date & Time,Total amount,Delivery option,Payment method
+0,118,in progress,2020-05-14 15:50:53.273417,9600.0,Takeaway,Cash
+1,498,in progress,2020-02-12 15:50:53.273417,4300.0,Takeaway,Cash
+2,348,Received,2020-03-19 15:50:53.273417,2000.0,Takeaway,Cash
+3,768,Received,2020-02-16 15:50:53.273417,13040.0,Takeaway,Cash
+4,210,in progress,2020-01-31 15:50:53.273417,3210.0,Takeaway,Cash
+5,445,Cancelled,2020-4-16 15:50:53.273417,8760.0,Takeaway,Cash
+6,321,in progress,2020-4-15 15:50:53.273417,2650.0,Takeaway,Cash
+7,765,in progress,2020-7-18 15:50:53.273417,3100.0,Takeaway,Cash
+8,327,in progress,2020-7-06 15:50:53.273417,7600.0,Takeaway,Cash
```

Comment:
I can see few records are on different date formats, because of which sorting of orders as per date is not proper.
Modify these dates in the DB and the use same kind of date-time format to avoid such errors.
![Screenshot 2020-07-30 at 12 27 32 AM](https://user-images.githubusercontent.com/54791364/88841714-f70c5b00-d1fb-11ea-9fc5-a74eb565038c.png)